tests/openadapt/test_scrub.py

```diff
@@ -0,0 +1,181 @@
+"""Module to test scrub.py"""
+from io import BytesIO
+import os
+from PIL import Image
+import pytesseract
+from openadapt.scrub import scrub, scrub_image
+
+
+def test_scrub_image_data() -> None:
+    """
+    Test is to be sure that the scrubbed image data is different
+    """
+    # Read test image data from file
+    test_image_path = "assets/test_scrub_image.png"
+    with open(test_image_path, "rb") as file:
+        test_image_data = file.read()
+
+    # Convert image data to PIL Image object
+    test_image = Image.open(BytesIO(test_image_data))
+
+    # Scrub the image
+    scrubbed_image = scrub_image(test_image)
+
+    # Save the scrubbed image data to a file
+    scrubbed_image_path = "scrubbed_image.png"
+    scrubbed_image.save(scrubbed_image_path)
+
+    # Load the scrubbed image from file for manual verification
+    scrubbed_image = Image.open(scrubbed_image_path)
+
+    # Perform OCR on the scrubbed image
+    ocr_text = pytesseract.image_to_string(scrubbed_image)
```

Comment:
As I understand it, this test is a smoke test to make sure that the module runs, as there is no automated analysis of the results.

For manual verification like you are doing here, there should be something like this at the bottom of `scrub.py`:

```
if __name__ == "__main__":
    fire.Fire(utils.get_functions(__name__))
```
utils.get_functions:
```
def get_functions(name):
    """
    Get a dictionary of function names to functions for all non-private functions

    Usage:

        if __name__ == "__main__":
            fire.Fire(utils.get_functions(__name__))

    Args:
        name: string containing name of module in sys.modules

    Returns:
        A dictionary of function names to functions.
    """
    functions = {}
    for name, obj in inspect.getmembers(sys.modules[name]):
        if inspect.isfunction(obj) and not name.startswith("_"):
            functions[name] = obj
    return functions
```

For this test, you can either:
1. remove this line and the one before it, and call it a "smoke" test
2. add some functionality to ensure that the output is as expected, e.g. with PIL by counting the number of pixels having the color of the mask and asserting it is e.g. 1/10th the total number of pixels in the image (or whatever is appropriate)

Reply:
Picked Option 2. Looked more robust.!